packages/jest-mock/src/index.ts

```diff
@@ -3,22 +3,33 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow
  */
 
-import type {Global} from 'types/Global';
-
-type Mock = any;
-export type MockFunctionMetadata = {
-  ref?: any,
-  members?: {[key: string]: MockFunctionMetadata},
-  mockImpl?: () => any,
-  name?: string,
-  refID?: string | number,
-  type?: string,
-  value?: any,
-  length?: number,
+type Global = NodeJS.Global;
```

Comment:
```suggestion
type Global = NodeJS.Global | Window;
```

Reply:
Makes sense, but:
> Property 'Object' does not exist on type 'Window'.ts(2339)

what do we do?

Reply:
Huh, guess we can't do it, then :(

Seems like wring types in the TS compiler?

![image](https://user-images.githubusercontent.com/1404810/52532229-c42d8900-2d21-11e9-8af7-53da8fbd4817.png)

![image](https://user-images.githubusercontent.com/1404810/52532231-c7c11000-2d21-11e9-8981-a9cbac520015.png)